src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1676,7 +1674,7 @@ protected override void InvokeDefaultAction(string path)
             ReturnContainers returnContainers)
         {
             if (tracker != null)
-                tracker.Visit(directory.FullName);
+                tracker.VisitPath(directory.FullName);
```

Comment:
Wrong pattern.
We can use `tracker?.VisitPath(directory.FullName); ` or `If (...) { ... }`.